lib/ansiblelint/runner.py

```diff
@@ -1,18 +1,28 @@
 """Runner implementation."""
 import logging
 import os
+import re
+import subprocess
 from dataclasses import dataclass
 from typing import TYPE_CHECKING, Any, FrozenSet, Generator, List, Optional, Set
 
 import ansiblelint.file_utils
 import ansiblelint.skip_utils
-import ansiblelint.utils
-from ansiblelint._internal.rules import LoadingFailureRule
+from ansiblelint._internal.rules import BaseRule, LoadingFailureRule, RuntimeErrorRule
 from ansiblelint.errors import MatchError
+from ansiblelint.rules.AnsibleSyntaxCheckRule import AnsibleSyntaxCheckRule
```

Comment:
Not relevant to this PR, but still: having files with camel-case names in the Python project is just plain bad. I had to have a look at what exactly is going on because this made no sense on the initial read.

Reply:
@tadeboro Shortly: historical reasons. As a note only rules are like this, all the other modules are lowercase. If you have time to look into a better solution I would not mind refactoring them.